frontend/src/components/util/routeInjector.tsx

```diff
@@ -57,18 +63,24 @@ export default function RouteInjector({ children }: any) {
         // You can not get the edition with the name in the path
         // -> You probably need to update the edition name because it was altered.
         // -> Should be handled by transformer, but we save guard here.
-        if (hadRouterError && contextEditionName && contextEditionName !== routerEditionName) {
-            replace({
+        // -> Not allowed since this is state controlled and we want hypermedia controlled.
+        /*
+        if (hadRouterError && contextEditionName && contextEditionName !== routerEditionName && false) {
+            routerAction({
                 query: { ...query, edition: contextEditionName },
             }).catch(console.log);
         }
+         */
 
         // You do not have a path edition set but your config edition is set and defined -> set path edition
-        if (contextEditionName && !routerEditionName) {
-            replace({
+        // -> Not allowed since the query start with and empty object, and you have no way as to know if you changed the edition.
```

Comment:
```suggestion
        // -> Not allowed since the query starts with an empty object and you have no way of knowing whether you changed the edition.
```